gui/src/pages/Process.svelte

```diff
@@ -81,37 +86,47 @@
               </tr>
               <tr>
                 <td class="label">CPU</td>
-                <td>{process.cpuPercent.toFixed(2)}%</td>
-                <td
-                  ><svg
+                <td>
+                  {#if process.cpuPercent}
+                    {process.cpuPercent.toFixed(2)}%
+                  {/if}
+                </td>
+                <td>
+                  <svg
                     bind:this={sparklineSvg}
                     class="sparkline"
                     width="100"
                     height="30"
                     stroke-width="3"
-                  /></td
-                >
+                  />
+                </td>
               </tr>
               <tr>
                 <td class="label">Resident Memory</td>
-                <td><BytesLabel value={process.residentMemory} /></td>
+                <td>
+                  {#if process.residentMemory}
```

Comment:
I added some conditional wrappers to fix all the typescript errors this file was causing, but perhaps this is not the ideal way to handle these errors and we could fix the problem upstream where `process` is defined. What do you think @brandonbloom ?

Reply:
What were the errors before? That process may have fields with null or undefined values? Seems reasonable to handle those conditionally.